plugins/labormanager/joblabormapper.cpp

```diff
@@ -334,6 +345,8 @@ class jlfunc_construct_bld : public jlfunc
             return construction_build_labor(b);
         }
         break;
+        case df::building_type::RoadPaved:
+            return df::unit_labor::BUILD_ROAD;
```

Comment:
I think this is the issue - paved roads require architects, which the old case handled. I'll see if that fixes it.